src/main/resources/db/changeLog.xml

```diff
@@ -143,4 +143,12 @@
             <column name="updatedBy" type="binary(16)"/>
         </createTable>
     </changeSet>
+
+    <changeSet id="addDraftColumnToVacationRequest" author="Veikko Vilpponen">
+        <addColumn tableName="VacationRequest">
```

Comment:
See earlier comments on this matter.

Hold on, how on earth is @VeikkoVilpponen authoring this? Considering he hasn't worked on the project for I don't know how long ago since. :)